libtenzir/builtins/operators/unordered.cpp

```diff
@@ -94,6 +96,25 @@ class plugin final : public virtual operator_plugin<unordered_operator> {
     }
     return std::make_unique<unordered_operator>(std::move(result.inner));
   }
+
+  auto make(invocation inv, session ctx) const
+    -> failure_or<operator_ptr> override {
+    auto pipe = std::optional<pipeline>{};
+    auto parser = argument_parser2::operator_(name()).positional("{ … }", pipe);
+    TRY(parser.parse(inv, ctx));
+    if (not pipe) {
+      const auto* pass_plugin = plugins::find<operator_factory_plugin>("pass");
+      TENZIR_ASSERT(pass_plugin);
+      TRY(auto pass,
+          pass_plugin->make(invocation{.self = inv.self, .args = {}}, ctx));
+      return std::make_unique<unordered_operator>(std::move(pass));
+    }
```

Comment:
We agreed to make this required instead of optional to incentivize using the operator correctly.

Reply:
Done